### ISSUE TITLE
average loss = nan

### ISSUE THREAD

**Issue by suryabhupa** [koraykv/unsup#38]
When training a convolutional autoencoder without the Hessian information, it looks like the average losses are equal to nan; what exactly does the Hessian information bring about?

**Comment by soumith** [comment]
If you are getting a nan without hessian info, then reduce the learning rate. The hessian info will make the learning rate parameter to be less relevant, as it tunes per-parameter learning rate according to the curvature.

**Comment by matsaragas** [comment]
Thanks @soumith I had the same problem. I reduced the learning rate and it seems that the error is minimized correctly.

**Comment by suryabhupa** [comment]
I never followed up on this -- it worked! Thanks @soumith. 👍
